lib/appCleanup.js

```diff
@@ -31,28 +33,50 @@ else {
 
 console.log(`🛁  Cleaning up ${appName}...`.bold);
 
-// remove public directory
+// check for public directory
 if (utils.fileExists(publicDir)) {
-  console.log(`🗑  Removing directory: ${publicDir}`);
-  rimraf.sync(publicDir);
+  rmDirs.push(publicDir);
+  console.log(`🔦  Found directory: ${publicDir}`);
 }
 
-// remove compiled js directory
+// check for compiled js directory
 if (utils.fileExists(compiledJsDir)) {
-  console.log(`🗑  Removing directory: ${compiledJsDir}`);
-  rimraf.sync(compiledJsDir);
+  rmDirs.push(compiledJsDir);
+  console.log(`🔦  Found directory: ${compiledJsDir}`);
 }
 
-// remove compiled css directory
-if (utils.fileExists(compiledCssDir)) {
-  console.log(`🗑  Removing directory: ${compiledCssDir}`);
-  rimraf.sync(compiledCssDir);
+// check for compiled css directory (if unique)
+if (compiledJsDir !== compiledCssDir) {
+  if (utils.fileExists(compiledCssDir)) {
+    rmDirs.push(compiledCssDir);
+    console.log(`🔦  Found directory: ${compiledCssDir}`);
+  }
 }
 
-// remove bundled js directory
+// check for bundled js directory
 if (utils.fileExists(bundledJsDir)) {
-  console.log(`🗑  Removing directory: ${bundledJsDir}`);
-  rimraf.sync(bundledJsDir);
+  rmDirs.push(bundledJsDir);
+  console.log(`🔦  Found directory: ${bundledJsDir}`);
 }
 
-console.log('✔️  Cleanup finished.'.bold);
+// if directories are found, prompt user before deletion
+if (rmDirs[0]) {
+  rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
+
+  rl.question(`❔  Do you want to remove ${rmDirs[1] ? 'these directories' : 'this directory'}? [Y/n]`.bold, (answer) => {
```

Comment:
make it `[y/N]` with N being default if the user hits enter and doesn't supply an answer (or a valid answer)